dashboard/config/scripts/test_levelgroup_activityguide.level_group

```diff
@@ -0,0 +1,9 @@
+name 'test-levelgroup-activityguide'
+title 'Activity Guide Levelgroup'
+submittable 'false'
+anonymous 'false'
+activity_guide_level 'true'
+
+page
+level 'test-single-multi-retriable-in-levelgroup'
+level 'test-free-response-retriable-in-levelgroup'
```

Comment:
This is of zero importance to me, and I would not suggest "fixing" this here, but fyi some programmers are sticklers about it and its not worth the energy to argue with them so I just comply: people have a weird thing about always ending files with a trailing newline/enter. That's what the red do not pass or whatever symbol github is showing below line 9 is about: its warning you that linting-variety programmers will want an extra newline/enter after the end of line 9.